packages/wrangler/turbo.json

```diff
@@ -70,6 +70,7 @@
 				"WRANGLER",
 				"WRANGLER_IMPORT",
 				"MINIFLARE_IMPORT",
+				"MINIFLARE_WORKERD_PATH",
```

Comment:
I don't think this belongs in this PR.

It's a nice addition and we do not want to revert this if we revert the PR.

Reply:
Actually it was already here on line 69